### DISCUSSION TITLE
Is it possible to embed an iframe to Web Assembly?

### DISCUSSION THREAD

**Discussion by cangunaydin** [AvaloniaUI/Avalonia#14968] [Q&A]
Hello,
I am trying to embed iframe with avalonia app. But i am always getting cross origin policy problem.
![image](https://github.com/AvaloniaUI/Avalonia/assets/3688369/10a577bc-b592-4140-add7-d72f5754ef13)
![image](https://github.com/AvaloniaUI/Avalonia/assets/3688369/90aa4e79-b6a8-4ce6-b445-177128a78e08)

probably i am missing sth over here. I looked at ControlCatalog sample it does the same that i am trying to do, so i copy paste that code and couldn't make it work still. This is a youtube embedded video so i suppose it should work. 
I think i need those headers from the server hosting the web assembly. is there a way to add those headers on debug mode? 
I am not so sure how dotnet is serving the wasm app, probably with nodejs?

**Comment by maxkatz6** [comment]
See https://stackoverflow.com/questions/51480196/youtube-embeded-video-error-no-access-control-allow-origin

**Reply by cangunaydin** [reply]
Hello Max,
I have tried but it doesn't work 
![image](https://github.com/AvaloniaUI/Avalonia/assets/3688369/3ea1a46e-9f02-4f5d-86af-55cbe3cf16e5)

if i create a new mvc project and embed the same there it works. 
I tried angular app it works over there as well. I think the problem is with wasm, or how the wasm is hosted. Or maybe with my settings but i don't think so. Couldn't find what is wrong.
